compiler/parser.cpp

```diff
@@ -1069,60 +1067,6 @@ setcaption(void) {
     pc_printf("Copyright (c) 2004-2018 AlliedModders LLC\n\n");
 }
 
-static void
-about(void) {
-    if (strlen(errfname) == 0) {
-        setcaption();
-        pc_printf("Usage:   spcomp <filename> [filename...] [options]\n\n");

```

Comment:
The old documentation has filenames first then options. The new usage output shows options first. I guess the old way around still works too?

Reply:
Yes.